docs/systemd.md

```diff
@@ -0,0 +1,80 @@
+## systemd cgroup driver
+
+By default, runc creates cgroups and sets cgroup limits on its own (this mode
+is known as "fs" cgroup driver). When `--systemd-cgroup` global option is given
+(as in e.g. `runc --systemd-cgroup run ...`), runc switches to "systemd" cgroup
+driver.
+
+In particular, on `run` or `create`, runc requests systemd to create a scope
+(a.k.a. transient unit), usually named like `runc-$ID.scope`, and puts
```

Comment:
Can we describe how we expect the colon separated string to be set in config.json for systemd cgroups?

Reply:
Good idea! Will do.

Reply:
done